IRremoteESP8266.h

```diff
@@ -32,6 +32,7 @@
 #define IRremote_h
 
 #include <stdint.h>
+#include "IRremoteInt.h"
```

Comment:
I should add, when I attempt to shard up the files per plan for v2.0 , I'll attempt to get to the root of this problem.